ibis/expr/operations/udf.py

```diff
@@ -116,7 +117,13 @@ def _make_node(
                 "__udf_namespace__": schema,
                 "__module__": fn.__module__,
                 "__func_name__": func_name,
-                "__full_name__": ".".join(filter(None, (schema, func_name))),
+                # not actually a table, but easier to quote individual namespace
+                # components this way
+                "__sql_name__": staticmethod(
```

Comment:
Please move this to the base sqlglot compiler

Reply:
Good call, that removes the `sqlglot` import from `operations/udf.py` :)

Reply:
Moved.